### PR TITLE
Update package documentation

### PR DESCRIPTION
**Pull request by zbiljic**
This change adds some additional documentation on several packages.

### CODE REVIEW

File: pkg/addressbook/addressbook.go [hunk 1]
```diff
@@ -33,21 +36,25 @@ type GetPutter interface {
 }
 
 type Getter interface {
+	// Get returns pointer to saved bzz.Address for requested overlay address.
 	Get(overlay swarm.Address) (addr *bzz.Address, err error)
 }
 
 type Putter interface {
+	// Put saves relation between peer overlay address, and bzz.Address address.
```
Comment (janos) [inline pkg/addressbook/addressbook.go:44]: I think that `,` is not needed.

File: pkg/addressbook/doc.go [hunk 2]
```diff
@@ -0,0 +1,12 @@
+// Copyright 2020 The Swarm Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+/*
+Package addressbook provides persisted mapping between overlay (topology) address
+and bzz.Address address, which contains underlay (physical) address.
```
Comment (janos) [inline pkg/addressbook/doc.go:7]: I would say that underlay contains both physical and p2p addresses.

**Review by janos — APPROVED** [review]
Great! My comments are very minor.

File: pkg/addressbook/doc.go (hunk 2 above)
Comment (zbiljic) [reply pkg/addressbook/doc.go:7]: My thinking for this instance was that that reader would follow `bzz.Address` implementation and figure out rest there.
Anyway, I update this comment according to comment.

**Review by janos — APPROVED** [review]
Thanks. Very nice.

File: pkg/keystore/keystore.go [hunk 3]
```diff
@@ -9,9 +9,16 @@ import (
...
 
+// ErrInvalidPassword is returned when the password for decrypting content where
+// private key is stored is not valid.
 var ErrInvalidPassword = errors.New("invalid password")
 
+// Service for managing keystore private keys.
 type Service interface {
+	// Key returns private key for specified name that was encrypted with
+	// provided password. If the private key does not exists it creates new one
+	// with name and password, and returns with created set to true.
```
Comment (zelig) [inline pkg/keystore/keystore.go:20]: // Key returns the private key for a  specified name that was encrypted with the
	// provided password. If the private key does not exists it creates a  new one
	// with a name and the password, and returns with `created` set to true.

File: pkg/keystore/mem/service.go [hunk 4]
```diff
@@ -15,11 +15,17 @@ import (
 
 var _ keystore.Service = (*Service)(nil)
 
+// Service is the memory-based keystore.Service implementation.
+//
+// Keys are stored in the in-memory map, where the key is the name of the
+// private key, and value is structure where actual private key and password are
+// stored.
```
Comment (zelig) [inline pkg/keystore/mem/service.go:22]: // Keys are stored in an in-memory map, where the key is the name of the
// private key, and the value is the structure where the actual private key and the password are
// stored.

File: pkg/keystore/test/test.go [hunk 5]
```diff
@@ -12,6 +12,8 @@ import (
 	"github.com/ethersphere/bee/pkg/keystore"
 )
 
+// Service is utility testing function that can be used to test implementations
+// of keystore.Service interface.
```
Comment (zelig) [inline pkg/keystore/test/test.go:16]: // Service is a utility testing function that can be used to test implementations
// of the keystore.Service interface.

File: pkg/tracing/doc.go [hunk 6]
```diff
@@ -0,0 +1,51 @@
+// Copyright 2020 The Swarm Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+/*
+Package tracing helps with propagation of tracing span through context in the
+system. It does this for operations contained to single node, as well as across
+nodes, by injecting special headers.
+
```
Comment (zelig) [inline pkg/tracing/doc.go:9]: Package tracing helps with the propagation of the tracing span through context in the
system. It does this for operations contained to single node, as well as across
nodes, by injecting special headers.

File: pkg/tracing/doc.go [hunk 7]
```diff
@@ -0,0 +1,51 @@
...
+To use tracing package Tracer instance must be created, which contains functions
+for starting new span contexes, injecting them in other data, and extracting
+active span them from context.
```
Comment (zelig) [inline pkg/tracing/doc.go:12]: To use the tracing package, a Tracer instance must be created, which contains functions
for starting new span contexts, injecting them in other data, and extracting the
active span them from the context.

File: pkg/tracing/doc.go [hunk 8]
```diff
@@ -0,0 +1,51 @@
...
+
+To use tracing package Tracer instance must be created:
```
Comment (zelig) [inline pkg/tracing/doc.go:14]: To use the tracing package a Tracer instance must be created:

File: pkg/tracing/doc.go [hunk 9]
```diff
@@ -0,0 +1,51 @@
...
+		ServiceName: "bee",
+	})
+	if err != nil {
+		// handle error
+	}
+	defer tracerCloser.Close()
+	// ...
+
+The tracer instance contains functions for starting new span contexes, injecting
+them in other data, and extracting active span them from context:
```
Comment (zelig) [inline pkg/tracing/doc.go:28]: The tracer instance contains functions for starting new span contexts, injecting
them in other data, and extracting the active span them from the context:

File: pkg/tracing/doc.go [hunk 10]
```diff
@@ -0,0 +1,51 @@
...
+
+	span, _, ctx := tracer.StartSpanFromContext(ctx, "operation-name", nil)
+
+Once the operation is finished, the open span should be finished:
+
+	span.Finish()
+
+The tracing package also provides function for creating logger which will inject
+"traceid" field entry to log line, which helps in finding out which log lines
+belong to specific trace.
```
Comment (zelig) [inline pkg/tracing/doc.go:38]: The tracing package also provides a function for creating  a  logger which will inject
a "traceid" field entry to the log line, which helps in finding out which log lines
belong to a specific trace.

File: pkg/tracing/doc.go [hunk 11]
```diff
@@ -0,0 +1,51 @@
...
+
+To create logger with trace just wrap existing logger:
```
Comment (zelig) [inline pkg/tracing/doc.go:40]: To create a logger with trace just wrap an existing logger

**Review by zelig — CHANGES_REQUESTED** [review]
the definite and indefinite articles are a bitch for slavic speakers but without them the text is very difficult to read.

**Review by zelig — APPROVED** [review]

**Review by acud — APPROVED** [review]
LGTM

**Pull request merged**